using-mobile-extensions/places/places-usage-reference.md

```diff
@@ -0,0 +1,336 @@
+# AEPPlaces Public APIs
+
+This document contains usage information for the public functions, classes, and enums in AEPPlaces.
+
+## Static Functions
+
+- [clear](#clear)
+- [extensionVersion](#extensionVersion)
+- [getCurrentPointsOfInterest](#getCurrentPointsOfInterest)
+- [getLastKnownLocation](#getLastKnownLocation)
+- [getNearbyPointsOfInterest](#getNearbyPointsOfInterest)
+- [processRegionEvent](#processRegionEvent)
+- [registerExtension](#registerExtension)
+- [setAuthorizationStatus](#setAuthorizationStatus)
+
+<hr />
+
+### clear
+
+Clears out the client-side data for Places in shared state, local storage, and in-memory.
+
+{% tabs %}
+{% tab title="Swift" %}
+<b>Signature:</b>
+```
+static func clear()
+```
+<b>Example Usage:</b>
+```
+Places.clear()
+```
+{% endtab %}
+{% tab title="Objective-c" %}
+<b>Signature:</b>
+```
++ (void) clear;
+```
+<b>Example Usage:</b>
+```
+[AEPMobilePlaces clear];
+```
+{% endtab %}
+{% endtabs %}
+
+<hr />
+
+### extensionVersion
+
+Returns the running version of the AEPPlaces extension.
+
+{% tabs %}
+{% tab title="Swift" %}
+<b>Signature:</b>
+```
+static var extensionVersion: String
+```
+<b>Example Usage:</b>
+```
+let placesVersion = Places.extensionVersion
+```
+{% endtab %}
+{% tab title="Objective-c" %}
+<b>Signature:</b>
+```
++ (nonnull NSString*) extensionVersion;
+```
+<b>Example Usage:</b>
+```
+NSString *placesVersion = [AEPMobilePlaces extensionVersion];
+```
+{% endtab %}
+{% endtabs %}
+
+<hr />
+
+### getCurrentPointsOfInterest
+
+Returns all Points of Interest (POI) of which the device is currently known to be within.
+
+{% tabs %}
+{% tab title="Swift" %}
+<b>Signature:</b>
+```
+static func getCurrentPointsOfInterest(_ closure: @escaping ([PointOfInterest]) -> Void)
+```
+<b>Example Usage:</b>
+```
+Places.getCurrentPointsOfInterest() { currentPois in
+    print("currentPois: \(currentPois)")
+}
+```
+{% endtab %}
+{% tab title="Objective-c" %}
+<b>Signature:</b>
+```
++ (void) getCurrentPointsOfInterest: ^(NSArray<AEPPlacesPoi*>* _Nonnull pois) closure;
+```
+<b>Example Usage:</b>
+```
+[AEPMobilePlaces getCurrentPointsOfInterest:^(NSArray<AEPPlacesPoi *> *pois) {
+    NSLog(@"currentPois: %@", pois);
+}];
+```
+{% endtab %}
+{% endtabs %}
+
+<hr />
+
+### getLastKnownLocation
+
+Returns the last latitude and longitude provided to the AEPPlaces Extension.
+
+If the Places Extension does not have a valid last known location for the user, the parameter passed in the closure will be `nil`. The CLLocation object returned by this method will only ever contain valid data for latitude and longitude, and is not meant to be used for plotting course, speed, altitude, etc.
```

Comment:
sure.  the `etc.` is because i didn't want to list out every property of the CLLocation object (https://developer.apple.com/documentation/corelocation/cllocation?language=objc#topics) 

should i name the whole list?

Reply:
how does this sound? 

```
The `CLLocation` object returned by this method will only contain a valid coordinate. Other properties on the `CLLocation` object should not be considered valid.
```

Reply:
Yeah, that would be good - and linking out to that list for further clarification would work really well IMO.